<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dequip</class>
 <widget class="QWidget" name="Dequip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1771</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1731</width>
     <height>771</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="requestButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>810</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发出请求</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
